.gitignore

```diff
@@ -4,6 +4,7 @@ obj
 *.mdb
 /GameData/FerramAerospaceResearch/Custom*.cfg
 /GameData/FerramAerospaceResearch/FARForceDataUpdate.cfg
+/note.txt
```

Comment:
No need to include this in PR

Reply:
Just need a few minutes to figure out how to do this :-)

Reply:
I just pushed the 'removed' note.txt to the source branch. I hope this is the Github way that I am supposed to solve the comment.